docs/_dg/ModelComponent.md

```diff
@@ -0,0 +1,15 @@
+!-- markdownlint-disable-file first-line-h1 --
+### Model component
+
```

Comment:
These lines are incorrect. The repeat header will be included into the DG. Also, the lint ignore statements should be comments.

```suggestion
<!-- markdownlint-disable-file first-line-h1 -->
```